catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/CswSourceConfiguration.java

```diff
@@ -92,7 +103,11 @@ public String getPassword() {
     }
 
     public void setPassword(String password) {
```

Comment:
The example you mentioned prevents a user from updating the password with an empty string.We should still allow a user to update it to empty string.

The reason for checking if the string is not empty in the first place is because the encryption service returns null if an empty string is passed in.